hudi/tests/docker/project/plugins.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
```

Comment:
What does this do?

Reply:
This helps build the scala app https://github.com/DataDog/integrations-core/pull/10155/files#diff-558779a11418b79421d64644b3117c55f6b84c19b8e177916625d1fbede3e82fR24